components/dashboard/package.json

```diff
@@ -41,7 +41,7 @@
   },
   "scripts": {
     "start": "react-scripts start",
-    "build": "react-scripts build",
+    "build": "node script.js && react-scripts build",
```

Comment:
Since script run before build , how do we expect ENV Overrides to happen to provide override paths ? Just a question

Reply:
I dont clearly understand the question.

If you mean to ask how to change the paths, then we can just change it in the script and rebuild the project. And all the changed env vars will be available.

Reply:
We need a mechanism for build. This is a public repo.

Reply:
There is a already a predefined build mechanism in the repo. Everything works out same as it used to

Reply:
We can use the public s3 urls for images in script.js . Or we can manully replace the icons in icons/public folder. And then build the dashboard component